packages/terra-clinical-result/src/ClinicalResult.module.scss

```
@@ -110,7 +110,11 @@
   .result-display-separator {
     color: var(--terra-clinical-result-display-separator-color);
     font-weight: var(--terra-clinical-result-display-separator-font-weight, normal);
-    margin-left: var(--terra-clinical-result-display-separator-margin-left, 0.25em); // Must use ems for font scaling
-    margin-right: var(--terra-clinical-result-display-separator-margin-right, 0.25em); // Must use ems for font scaling
+    margin-left: var(--terra-clinical-result-display-separator-margin-left, 0.07em); // Must use ems for font scaling
+    margin-right: var(--terra-clinical-result-display-separator-margin-right, 0.107em); // Must use ems for font scaling
+  }
+
+  .diastolic-intrepretation {
```

Founds a spelling typo: 
```suggestion
  .diastolic-interpretation {
```

also here: [ClinicalResultBloodPressure.jsx L216](https://github.com/cerner/terra-clinical/pull/637/files#diff-13926e711e02256769a4612c0d911268R216), can fix with @emilyrohrbough comment https://github.com/cerner/terra-clinical/pull/637/files#r399484300

---

Fixed here https://github.com/cerner/terra-clinical/pull/637/commits/788a6eed719217b512d85dbc8c1fa1c37d747e50#diff-4cc08ccd9b257901c471e103f66b80a6R117